templates/qa_zre/templates.yaml

```diff
@@ -0,0 +1,50 @@
+dataset: qa_zre
+templates:
+  2d6b6ec6-4cba-4a07-a0d1-f6b7cb103281: !Template
+    id: 2d6b6ec6-4cba-4a07-a0d1-f6b7cb103281
+    jinja: 'Extract the appropriate relation from the following question
```

Comment:
Maybe include a variant of this task where the XXX is filled in with the subject? It's a valid task either way.